README.md

```diff
@@ -0,0 +1,175 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>VET CLINIC DATABASE</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+<details>
+  <summary>
+    <h1>📗 Table of Contents</h1>
+  </summary>
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+</details>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Space Travelers' Hub <a name="about-project"></a>
+
```

Comment:
- [ ] You are working on the Vet clinic database: create animals table milestone but on your README it is written Space Travelers' Hub. Please fix that.